examples/default/realign.json

```diff
@@ -0,0 +1,37 @@
+{
+    "@context": 
+      {
+        "bids" : "https://some/url/to/bids_context.jsonld",
+        "exp_dir" : {"@id" : "$HOME/nidm-results-examples/spm-default/ds011/"}
+      },
+    "@id": "http://example.org/ds00000X",
+    "generatedAt": "2020-03-10T10:00:00",
+    "wasGeneratedBy": {
+        "@id": "INRIA",
+        "@type": "Project",
+        "startedAt": "2016-09-01T10:00:00",
+        "wasAssociatedWith": { "@id": "NIH",
+                               "@type": "Organization",
+                               "hadRole": "Funding"
+                             }
+      },
+    "@graph": [
+      {
+        "@id" : "realign",
+        "@type" : "bids:bids_activity",
+        "wasAssociatedWith" : "bids:spmspatial.realign.estwrite",
```

Comment:
For the model to be scalable, we need a lightweigth way to define new agents. How would you envision this?

Reply:
Depends on what the `bids` namespace if referring to.
If agents are defined in Bids-terms, then people maintaining Bids-terms will be in charge of adding new Agents to the list.

But perhaps that's not what you meant. By "lightweight" do you mean that a anyone should be able to define a new "custom" agent ??

Reply:
Yes the latter is what I was getting at... How do we make easy for people to "declare" new agents that corresponds to their tools...

Reply:
OK so the W3C prov namespace gives access to `prov:Agent`. We can simply use this class and its subclasses in our examples, and show the way.

Reply:
Yes we could also add an attribute that store the name of the agent (as a string) to be more specific. This would make the model less constrained (in that any string can be used to describe an agent). How is that?

Reply:
OK so back to my first implementation